net/local/local_sendmsg.c

```diff
@@ -417,14 +426,7 @@ ssize_t local_sendmsg(FAR struct socket *psock, FAR struct msghdr *msg,
   if (len < 0 && count > 0)
     {
       net_lock();
-
-      while (count-- > 0)
-        {
-          file_close(conn->lc_cfps[--conn->lc_cfpcount]);
-          kmm_free(conn->lc_cfps[conn->lc_cfpcount]);
-          conn->lc_cfps[conn->lc_cfpcount] = NULL;
-        }
-
+      local_freectl(conn, count);
```

Comment:
Note: We need to free `conn` with `count` here, but `peer` with `i` above.

Reply:
No, the old code free the wrong array.

Reply:
OK, I see.